x86_64/python-fenics-ffcx/cactus.yaml

```diff
@@ -0,0 +1,9 @@
+nvchecker:
+  - source: aur
+    aur:
```

Comment:
Should also have the python alias upstream. And so do the other python packages.

Reply:
Thanks, now include python alias.